overrides-locales/malte.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Where are locales actually overwritten? I could not find anything related to that.

Reply:
Will be done in later issues in the app repos: https://issues.integreat-app.de/browse/NATIVE-559 and https://issues.integreat-app.de/browse/WEBAPP-586